cmd/main.go

```diff
@@ -34,8 +34,10 @@ func main() {
 		cli.WithDefaultPlugins(
 			&pluginv2.Plugin{},
 		),
-		cli.WithExtraCommands(
+		cli.WithExtraCommandsForV2(
```

Comment:
IMO we need to move for v2 and v3 in this case. I do not think we can do this change `WithExtraCommandsForV2 ` in this case. I mean, we should not change the behaviour. We would just making the edit be an plugin for v2,v3

Reply:
Got it.. So i will move the edit command to plugin for the v2 as well. In that case, I need to remove the newEditCmd() from the cli.WithExtraCommands.

Reply:
Done